concepticondata/conceptlists/Urban-2016-160.tsv

```diff
@@ -0,0 +1,161 @@
+Urban-2016-160-65	65	thunder	Nature-related and topological concepts	1150	THUNDER
+Urban-2016-160-66	66	tree	Nature-related and topological concepts	906	TREE
+Urban-2016-160-67	67	valley	Nature-related and topological concepts	921	VALLEY
+Urban-2016-160-68	68	volcano	Nature-related and topological concepts	2291	VOLCANO
+Urban-2016-160-69	69	waterfall	Nature-related and topological concepts	943	WATERFALL
+Urban-2016-160-70	70	wave	Nature-related and topological concepts	978	WAVE
+Urban-2016-160-71	71	wax	Nature-related and topological concepts	1931	BEESWAX
+Urban-2016-160-72	72	whirlpool	Nature-related and topological concepts	1982	WHIRLPOOL
+Urban-2016-160-73	73	airplane	Artifacts	1597	AIRPLANE
+Urban-2016-160-74	74	ball	Artifacts	1845	BALL
+Urban-2016-160-75	75	bed	Artifacts	1663	BED
+Urban-2016-160-76	76	belt	Artifacts	772	BELT
+Urban-2016-160-77	77	boat	Artifacts	1844	BOAT
+Urban-2016-160-78	78	car	Artifacts	1111	CAR
+Urban-2016-160-79	79	chair	Artifacts	1476	CHAIR
+Urban-2016-160-80	80	clock	Artifacts	1323	CLOCK
+Urban-2016-160-81	81	glasses	Artifacts	2292	GLASSES
+Urban-2016-160-82	82	house	Artifacts	1252	HOUSE
+Urban-2016-160-83	83	key	Artifacts	1472	KEY
+Urban-2016-160-84	84	knife	Artifacts	1352	KNIFE
+Urban-2016-160-85	85	ladder	Artifacts	1464	LADDER
+Urban-2016-160-86	86	mirror	Artifacts	1171	MIRROR
+Urban-2016-160-87	87	needle	Artifacts	1382	NEEDLE (FOR SEWING)
+Urban-2016-160-88	88	paper	Artifacts	648	PAPER
+Urban-2016-160-89	89	pen	Artifacts	794	PEN
+Urban-2016-160-90	90	rope	Artifacts	1218	ROPE
+Urban-2016-160-91	91	scissors	Artifacts	1328	SCISSORS
+Urban-2016-160-92	92	shoe	Artifacts	1381	SHOE
+Urban-2016-160-93	93	road/street/way	Artifacts	2457	PATH OR ROAD
+Urban-2016-160-94	94	table	Artifacts	1775	TABLE
+Urban-2016-160-95	95	toilet	Artifacts	219	TOILET
+Urban-2016-160-96	96	train	Artifacts	900	TRAIN
+Urban-2016-160-97	97	weapon	Artifacts	951	WEAPONS
+Urban-2016-160-98	98	window	Artifacts	1375	WINDOW
+Urban-2016-160-99	99	Adam‘s apple	Body Parts and Body Fluids	802	ADAM'S APPLE
+Urban-2016-160-100	100	ankle	Body Parts and Body Fluids	803	ANKLE
+Urban-2016-160-101	101	beard	Body Parts and Body Fluids	678	BEARD
+Urban-2016-160-102	102	belly/stomach	Body Parts and Body Fluids	2114	BELLY OR STOMACH
+Urban-2016-160-103	103	bladder	Body Parts and Body Fluids	1746	BLADDER
+Urban-2016-160-104	104	blood	Body Parts and Body Fluids	946	BLOOD
+Urban-2016-160-105	105	bone	Body Parts and Body Fluids	1394	BONE
+Urban-2016-160-106	106	brain	Body Parts and Body Fluids	1583	BRAIN
+Urban-2016-160-107	107	breast	Body Parts and Body Fluids	1402	BREAST
+Urban-2016-160-108	108	buttocks	Body Parts and Body Fluids	834	BUTTOCKS
+Urban-2016-160-109	109	calf	Body Parts and Body Fluids	498	CALF OF LEG
+Urban-2016-160-110	110	cheek	Body Parts and Body Fluids	1730	CHEEK
+Urban-2016-160-111	111	chin	Body Parts and Body Fluids	1510	CHIN
+Urban-2016-160-112	112	eyeball	Body Parts and Body Fluids		
```

Comment:
Other lists including (Urban-2011-160-112) mapped "eyeball" to EYE.

Reply:
Hale-1973-1798-122 eyeball is also mapped to EYE. Perhaps, this instance can be linked to EYE too also because it doesn't contrast in this list with anything other eye-related, and the concept of EYE seems logical among other body parts concepts.